cloud-nio/cloud-nio-spi/src/main/scala/cloud/nio/spi/UnixPath.scala

```diff
@@ -67,7 +67,7 @@ final private[spi] case class UnixPath(path: String) extends CharSequence {
 
   def isAbsolute: Boolean = UnixPath.isAbsolute(path)
 
-  def isEmpty: Boolean = path.isEmpty
```

Comment:
This is a name collision new in 17.

The initial compile error is that it needs an `override`. Adding the `override` results in a second error saying it overrides nothing!